contracts/strategies/RFPStrategy.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IAllo} from "../core/IAllo.sol";
+import {IRegistry} from "../core/IRegistry.sol";
+import {BaseStrategy} from "./BaseStrategy.sol";
+import {Metadata} from "../core/libraries/Metadata.sol";
+import {ReentrancyGuard} from "@openzeppelin/security/ReentrancyGuard.sol";
+
+contract RFPStrategy is BaseStrategy, ReentrancyGuard {
+    /// ================================
+    /// ========== Storage =============
+    /// ================================
+
+    /// @notice Struct to hold details of an recipient
+    struct Recipient {
+        address recipientAddress;
+        uint256 proposalBid;
+        RecipientStatus recipientStatus;
+    }
```

Comment:
Don't we need a recipientID and a payout address here?

Reply:
recipientId is not required, since the relation is stored in
```solidity
 /// @notice recipientId -> Recipient
 mapping(address => Recipient) public recipients;
```


The payout address is the same as `recipientAddress`